docs/quickstart.rst

```diff
@@ -51,19 +51,14 @@ Put a list of filters into `~/.config/afew/config`:
     [ArchiveSentMailsFilter]
     [InboxFilter]
 
-And create a post-new hook for notmuch.
+And create a `post-new` hook for notmuch to call afew:
 
 .. code-block:: sh
 
-    $ mkdir -p path/to/maildir/.notmuch/hooks
-    $ touch path/to/maildir/.notmuch/hooks/post-new
-
-Then edit the `post-new` file to contain:
-
-.. code-block:: sh
-
-    #!/bin/sh
-    $HOME/.local/bin/afew --tag --new
+    $ notmuchdir=path/to/maildir/.notmuch
+    $ mkdir -p "$notmuchdir/hooks"
+    $ printf > "$notmuchdir/hooks/post-new" '#!/bin/sh\n$HOME/.local/bin/afew --tag --new\n'
```

Comment:
I would prefer a more portable shebang like '#!/usr/bin/env bash',
but this change would require to update everywhere, so it can be done later.

Reply:
`/bin/sh` should also exist, and is preferrable if we don't rely on any bashisms.